docs/standard/linq/control-type-projection.md

```diff
@@ -0,0 +1,135 @@
+    {
+        CustomerID = customerID;
+        CompanyName = companyName;
+        ContactName = contactName;
+    }
+
+    public override string ToString()
+    {
+        return $"{this.customerID}:{this.companyName}:{this.contactName}";
+    }
+}
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        XElement custOrd = XElement.Load("CustomersOrders.xml");
+        IEnumerable<Customer> custList =
+            from el in custOrd.Element("Customers").Elements("Customer")
+            select new Customer(
+                (string)el.Attribute("CustomerID"),
+                (string)el.Element("CompanyName"),
+                (string)el.Element("ContactName")
+            );
+        foreach (Customer cust in custList)
+            Console.WriteLine(cust);
+    }
+
+
+}
+```
+
+```vb
+Public Class Customer
+    Private customerIDValue As String
+    Public Property CustomerID() As String
+        Get
+            Return customerIDValue
+        End Get
+        Set(ByVal value As String)
+            customerIDValue = value
+        End Set
+    End Property
+
+    Private companyNameValue As String
+    Public Property CompanyName() As String
+        Get
+            Return companyNameValue
+        End Get
+        Set(ByVal value As String)
+            companyNameValue = value
+        End Set
+    End Property
+
+    Private contactNameValue As String
+    Public Property ContactName() As String
+        Get
+            Return contactNameValue
+        End Get
+        Set(ByVal value As String)
+            contactNameValue = value
+        End Set
+    End Property
+
+    Public Sub New(ByVal customerID As String, _
+                   ByVal companyName As String, _
+                   ByVal contactName As String)
+        CustomerIDValue = customerID
+        CompanyNameValue = companyName
+        ContactNameValue = contactName
+    End Sub
+
+    Public Overrides Function ToString() As String
+        Return String.Format("{0}:{1}:{2}", Me.CustomerID, Me.CompanyName, Me.ContactName)
+    End Function
+End Class
+
+Sub Main()
+    Dim custOrd As XElement = XElement.Load("CustomersOrders.xml")
+    Dim custList As IEnumerable(Of Customer) = _
+        From el In custOrd.<Customers>.<Customer> _
+        Select New Customer( _
+            el.@<CustomerID>, _
+            el.<CompanyName>.Value, _
+            el.<ContactName>.Value _
+        )
+    For Each cust In custList
+        Console.WriteLine(cust)
+    Next
+End Sub
+```
+
+The example produces this output:
+
+```output
+GREAL:Great Lakes Food Market:Howard Snyder
+HUNGC:Hungry Coyote Import Store:Yoshi Latimer
+LAZYK:Lazy K Kountry Store:John Steel
+LETSS:Let's Stop N Shop:Jaime Yorres
+```
+
+## See also
+
+- <xref:System.Linq.Enumerable.Select%2A>
+- [Projections and Transformations (LINQ to XML) (Visual Basic)](../../visual-basic/programming-guide/concepts/linq/projections-and-transformations-linq-to-xml.md)
```

Comment:
I imagine this article would be moved to this folder too, right?

Reply:
It just becomes a parent node. You make a good point, though. That does happen in this part of the project, and I could deal with it now.

Reply:
Up to you. We can merge as-is and then once you redirect, you'd have to change these links.